test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Internal/PagedBufferedTextWriterTest.cs

```diff
@@ -279,5 +339,20 @@ public override void Return(char[] buffer, bool clearArray = false)
                 Returned.Add(buffer);
             }
         }
+
+        private class RentMoreArrayPool : ArrayPool<char>
+        {
+            public IList<char[]> Returned { get; } = new List<char[]>();
+
+            public override char[] Rent(int minimumLength)
+            {
+                return new char[2 * minimumLength];
+            }
+
+            public override void Return(char[] buffer, bool clearArray = false)
+            {
+                Returned.Add(buffer);
```

Comment:
Not using `Returned` in new test that uses this. But seemed better to be consistent w/ the other test `ArrayPool<T>` here.